fixtures/gtfs2ntfs/minimal_with_config/input/calendar_dates.txt

```diff
@@ -0,0 +1,6 @@
+service_id,date,exception_type
```

Comment:
as it's a test for feed infos u can remove all the unnecessary data in calendar_dates, routes, trips, stop_times, stops to keep the fixtures files light

Reply:
Ok, done